packages/client/src/utils/validate.ts

```diff
@@ -653,7 +653,7 @@ export const validIDNumber =
 export const duplicateIDNumber =
   (fieldToDuplicateCheck: string): Validation =>
   (value: IFormFieldValue, drafts) => {
-    const valueToCheck = _.get(drafts, fieldToDuplicateCheck)
+    const valueToCheck = _.get(drafts, fieldToDuplicateCheck.split('.'))
```

Comment:
curious, why does this fix it? shouldn't lodash `get` support `x.y.z` syntax? https://lodash.com/docs/4.17.15#get